solr/modules/opentelemetry/src/test-files/solr/solr.xml

```diff
@@ -34,8 +34,6 @@
     <int name="connTimeout">${connTimeout:15000}</int>
   </shardHandlerFactory>
 
-  <tracerConfig name="tracerConfig" class="org.apache.solr.opentelemetry.CustomTestOtelTracerConfigurator" />
```

Comment:
Perhaps this class could return but only for a test explicitly testing that tracerConfig works and in a separate solr.xml for exactly that test as well (not for the other tracing tests).  Not to be used for other tests or trying to collect spans for examination.